test/upgradeability/OwnedUpgradeabilityProxy.test.js

```diff
@@ -249,22 +251,22 @@ contract('OwnedUpgradeabilityProxy', ([_, owner, anotherAccount]) => {
       const newOwner = anotherAccount
 
       describe('when the sender is the owner', function () {
-        const from = owner
-
-        it('transfers the ownership', async function () {
-          await this.proxy.transferProxyOwnership(newOwner, { from })
+        beforeEach('transferring', async function () {
+          const { logs } = await this.proxy.transferProxyOwnership(newOwner, { from: owner })
+          this.logs = logs
+        })
 
-          const proxyOwner = await this.proxy.proxyOwner()
+        it('assigns new proxy owner', async function () {
+          // [TODO] maybe we should test this by reading from storage
```

Comment:
it would be too implementative in this context IMO... we are already checking storage works as expected

Reply:
if you agree please remove the `TODO` note or add an issue to tackle it